index.html

```diff
@@ -219,6 +219,36 @@ <h4>Important caveat</h4>
     </div>
     <!-- End Basic rewrap command -->
 
+    <!-- BWF -->
+    <label class="recipe" for="bwf">Convert to (or create) Broadcast WAV</label>
+    <input type="checkbox" id="bwf">
+    <div class="hiding">
+      <h5>Generate Broadcast WAV</h5>
+      <p><code>ffmpeg -i <em>input_file.wav</em> -c copy -write_bext 1 -metadata field_name='Content' <em>output_file.wav</em></code></p>
+      <p>This command will create a write a file in the Broadcast WAV (BWF) containing a BEXT chunk with related metadata.</p>
```

Comment:
I think this sentence needs to be rewritten with the word format added and "a write" removed?

Reply:
thanks for catching that @kieranjol - I changed my mind on that sentence like three times and clearly subconsciously split the difference!